Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -93,42 +93,32 @@ public AuditRepositoryOrchImpl() {
      */
     public AcknowledgementType logAudit(LogEventSecureRequestType mess, AssertionType assertion) {
```

Comment:
Noticed something odd on this class -- this is the only file your formatter didn't apply to.  Can you take a look at this one particular file as a whole, and make sure all the changes you intended are here?